src/NuGet.Services.BasicSearch/ConfigurationService.cs

```diff
@@ -35,27 +59,40 @@ public bool TryGet(string key, out string value)
                 {
                     var cloudKey = key.Replace(':', '-'); // no ':' supported in cloud services
                     value = SafeRoleEnvironment.GetConfigurationSettingValue(cloudKey);
-                    return true;
+                    setValue = true;
```

Comment:
Can we rename it to `hasSetValue` or something like that? `setValue` got my brain into thinking that the variable means we have to set a value later on.